### ISSUE TITLE
v1.7.2-rc6 bug with sphinx role without link

### ISSUE THREAD

**Issue by AntoineD** [PyCQA/docformatter#229]
With v1.7.2-rc6 and the following example that has a `:meth:` role without link (the `!` before `X`):
```python
class C:
    """CC.

    :meth:`!X`
    """
```
produces the bad change where a blank space is introduced after the second `:`:
```python
class C:
    """CC.

    :meth: `!X`
    """
```

Curiously, this does not happen when the first sentence has only one character!
```python
class C:
    """C.

    :meth:`!X`
    """
```

**Comment by weibullguy** [comment]
@AntoineD should be fixed in v1.7.2-rc7.

**Comment by AntoineD** [comment]
Thank you.
